### ISSUE TITLE
Where possible the balance_updates table should include operation IDs

### ISSUE THREAD

**Issue by vishakh** [Cryptonomic/Conseil#471]
There is no user-friendly way to link `balance_updates` records to specific operations. Where possible, the table should include a value for an `operation_group_hash` column. 

This should apply to records whose source is either `operation` or `operation_result`.

**Comment by ivanopagano** [comment]
we actually have column referencing either the block hash (for updates on the block) and operation id (for updates on operations)

```scala
  /** Database column source SqlType(varchar) */
  val source: Rep[String] = column[String]("source")
  /** Database column source_id SqlType(int4), Default(None) */
  val sourceId: Rep[Option[Int]] = column[Option[Int]]("source_id", O.Default(None))
  /** Database column source_hash SqlType(varchar), Default(None) */
  val sourceHash: Rep[Option[String]] = column[Option[String]]("source_hash", O.Default(None))
 ```
What you see here is an excerpt from the slick table definition.

The source column is the discriminator, and currently can be valued as
- "block"
- "operation"

Depending on the actual source, one of the alternative reference columns is populated
- sourceHash (for block updates, referencing the blocks table)
- sourceId (for operation updates, referencing the operations table)

I don't know if this solves the issue, or if additional references are still needed

@vishakh @anonymoussprocket @ba205 take a look at this ☝️

**Comment by ivanopagano** [comment]
Closed by #536 ?

@anonymoussprocket @ba205
